src/pybind/mgr/dashboard/frontend/cypress.json

```diff
@@ -7,6 +7,7 @@
   "supportFile": "cypress/support/index.ts",
   "video": false,
   "defaultCommandTimeout": 120000,
+  "responseTimeout": 45000,
```

Comment:
Are we sure we need to increase the response time from 30 secs to 45 secs? Are we talking about HTTP requests? If that's the case I feel that the issue will be somewhere else :/

Reply:
> Are we sure we need to increase the response time from 30 secs to 45 secs? Are we talking about HTTP requests? If that's the case I feel that the issue will be somewhere else :/

I followed this https://github.com/cypress-io/cypress/issues/7062#issuecomment-750030997 and I think it's same issue for us. Somehow `cy.request()` is taking some more time in cypress browser env so `cy.visit()` after it is failing to load(also I tried removing that afterEach hook(which isn't consider best practice) but it's somehow broke some tests).